packages/sanity/src/core/form/inputs/DateInputs/base/DateTimeInput.tsx

```diff
@@ -74,13 +76,14 @@ export const DateTimeInput = forwardRef(function DateTimeInput(
   const suffix = (
     <Flex style={{padding: '5px'}}>
       <Button
+        aria-label={calendarLabels.ariaLabel}
         ref={buttonRef}
         icon={CalendarIcon}
         mode="bleed"
         onClick={handleClick}
         style={{display: 'block'}}
         data-testid="select-date-button"
-        tooltipProps={{content: 'Select date'}}
+        tooltipProps={{content: t('inputs.datetime.tooltip-content')}}
```

Comment:
Sorry for the late review here @ninaandal 
Wondering if this should also be part of the calendar labels, given previous comment https://github.com/sanity-io/sanity/pull/6952/files#r1644370958
There are already other keys in the calendar labels like `selectHour` and `selectMinute`